src/core/core.scale.js

```diff
@@ -704,7 +715,7 @@ module.exports = function(Chart) {
 					var isLeft = options.position === 'left';
 					scaleLabelX = isLeft ? me.left + (scaleLabelFont.size / 2) : me.right - (scaleLabelFont.size / 2);
 					scaleLabelY = me.top + ((me.bottom - me.top) / 2);
-					rotation = isLeft ? -0.5 * Math.PI : 0.5 * Math.PI;
+					rotation = -0.5 * Math.PI;
```

Comment:
The reason I wrote it the current way is that to me it felt more natural to turn my head to the right on the right side and to the left on the left side. 

I don't know what most chart packages do though. If everyone else has the proposed rotation then we can change it.